src/applications/vaos/utils/constants.js

```diff
@@ -96,6 +96,69 @@ export const TYPES_OF_CARE = [
   },
 ];
 
+export const LANGUAGES = [
+  {
+    id: 'english',
+    text: 'English',
+    value: 'English',
+  },
```

Comment:
Languages are copied from `preferred-provider.json` in `var-web`. uiSchema is only using `text` and `value` currently, but left the `id` property in there in case that is what needs to be submitted.  One of these fields may be removable.